### PR TITLE
egress documentation and scripts

### PR DESCRIPTION
**Pull request by cantsin**
## Description

Document the egress control process.

### CODE REVIEW

File: bin/run.sh [hunk 1]
```diff
@@ -44,5 +44,10 @@ echo "${DEPLOYMENT_DESCRIPTION}"
 # Record deployment using the New Relic Python Admin CLI
 newrelic-admin record-deploy "${NEW_RELIC_CONFIG_FILE}" "${DEPLOYMENT_DESCRIPTION}"
 
+# make sure python accepts the cloud.gov certificate authority file for proxy
```
Comment (cantsin) [inline bin/run.sh:47]: alternatively we could implement this as a cloud.gov [app task](https://docs.cloudfoundry.org/devguide/using-tasks.html) but I think it's a bit cleaner to embed in `run.sh` -- but open to discussion on this.

Comment (kfoley-18F) [reply bin/run.sh:47]: I agree with this choice. I suspect it also makes app backup+recovery clearer if there's an outage and we need to reconstruct Tock.

**Review by kfoley-18F — APPROVED** [review]
👏 👏 👏 
Thank you!!

**Comment by edwintorres** [comment]
This part of tock is new to me, but I read the documentation and it's clear. 🚀

**Comment by cantsin** [comment]
I'm closing this for now since we need to re-evaluate how we approach egress vis a vis staging Tock (not the staging environment, but how to actually get Tock started).

**Pull request closed without merging**